lib/file-helper.unit.test.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env jest */
+
+const fileHelper = require('../lib/file-helper')
+
+describe('getComponentData', () => {
+  test('returns an error if unable to load component data', () => {
```

Comment:
Minor, but is there a reason we're using `test` here when we use `it` everywhere else?

Reply:
Good point 🤔 There's a bit of info on [the difference here](https://stackoverflow.com/a/56072272/1669682), but I think `it` makes the most sense in terms of 1) what we're doing here, and 2) to be consistent with our other tests. 

I think the only reason it's crept in is because it's what [Jest uses in it's Getting Started guide](https://jestjs.io/docs/getting-started). Will amend now!